library/include/ck/library/reference_tensor_operation/cpu/reference_gemm_bias_transpose.hpp

```diff
@@ -0,0 +1,140 @@
+#pragma once
```

Comment:
I think we don't need a reference operator for every DeviceOp. We just need to have reference op for GEMM/Conv.

Here is an example

https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/56adf7e9cc4fcf6592151281a727e96b625bc54f/example/04_gemm_add_add_fastgelu/gemm_add_add_fastgelu_xdl_fp16.cpp#L212-L237

Reply:
Agree. Let us leave it to the task of unifying Gemm kernels.